### DISCUSSION TITLE
Best EOS for liq-vap flash for hydrocarbon mixture

### DISCUSSION THREAD

**Discussion by marcotognotti** [usnistgov/REFPROP-issues#623] [Q&A]
In a mixture of nitrogen, methane, ethylene, propane and isopentane at ambient temperature and low pressure (15 bar), which is the best model to evaluate the liquid-vapor phase fractions? Since using Peng-Robinson, GERG2008 and Refprop I'm getting some considerable differences.

**Comment by ianhbell** [comment]
Generally, either the GERG-2008 model, or the higher accuracy pure fluid models in REFPROP with the same mixing models as in GERG-2008 should be good choices. You can also use GERG-2008 in REFPROP.  It should hopefully be the case that the accuracy would go like this:

REFPROP (high-accuracy pure fluid EOS plus the departure functions and BIP from GERG-2008) > GERG-2008 > Peng-Robinson

It could also be the case that calculations fail, but you should get error codes if this happens.
